includes/core/classes/class-assets.php

```diff
@@ -121,6 +121,24 @@ public function add_global_object(): void {
 	 */
 	public function enqueue_scripts(): void {
 		wp_enqueue_style( 'dashicons' );
+		if ( has_block( 'gatherpress/venue' ) || 'gatherpress_venue' === get_post_type() ) {
```

Comment:
Why do we need the second part of this conditional `'gatherpress_venue' === get_post_type()`. Wouldn't this always be for the venue block (which also exists in the `gatherpress_venue` post type?

Reply:
@mauteri I don't know why we need it. But we seem to. 

When I add a new venue and the address input field is blank, the `has_block( 'gatherpress/venue' )` call is returning false. I don't know why that is the case. I look at the code editor and see the the block is there.

So without that conditional when you add a new venue, you will get fatal errors or a blank map because those assets do not load until that block has an address attached to it. I would need more time to take a deep dive to see why that is the case. But adding the extra conditional is certainly not harming anything.

Reply:
@jmarx We don't need this loaded for the venue block in a single event? That also loads a map.